oss-fuzz.sh

```diff
@@ -172,6 +176,10 @@ compile_fuzzer github.com/ethereum/go-ethereum/tests/fuzzers/bls12381 \
   FuzzG2Add fuzz_g2_add \
   $repo/tests/fuzzers/bls12381/bls12381_test.go
 
+compile_fuzzer github.com/ethereum/go-ethereum/tests/fuzzers/bls12381 \
+  FuzzCrossG2Mul fuzz_cross_g2_mul\
```

Comment:
This might not work, since `FuzzCrossG2Mul` is a prefix of `FuzzCrossG2MultiExp` 
Go complains on my machine about it:
```
go test --fuzz FuzzCrossG1Mul
testing: will not fuzz, -fuzz matches more than one fuzz test: [FuzzCrossG1MultiExp FuzzCrossG1Mul]
FAIL
exit status 1
``` 
The way I found to fix it, was to specify `FuzzCrossG1Mul.` with a trailing dot.
I don't how if the compile_fuzzer script will work with this, but I think we should give it a try like it is right now and fix it up if necessary

Reply:
I think `FuzzCrossG1Mul$` is another to way to make it work. However, we can try it as is. The compilescript passes it to gofuzz-shim. Seems to me that it's parsed literally, not regexpy: https://github.com/holiman/gofuzz-shim/blob/main/main.go#L95